keywording/text.xml

```diff
@@ -245,9 +245,12 @@ the arch teams are happy for help, so long as they know what is going on.
 
 <p>
 In order to request stabilization of an ebuild, file a bug to the package's
-maintainer (which may be yourself), and list all secondary maintainers
-in the bug's CC. When the maintainers consider the ebuild to be ready for
-stabilization, they will add the relevant architecture teams to the CC list.
+maintainer (which may be yourself) in the Stabilization component, and list
+all secondary maintainers in the bug's CC. When the maintainers consider the
+ebuild to be ready for stabilization, they will add the relevant architecture
+teams to the CC list, either manually, or using
+<uri link="https://github.com/mgorny/nattka/#filing-keywordingstabilization-bugs">
+NATTkA</uri>.
```

Comment:
Do we have any Gentoo hosted documentation for this, e.g. on the wiki?

Reply:
The only reference I found in wiki to it is at https://wiki.gentoo.org/wiki/Proxied_Maintainer_FAQ#When_and_how_to_stabilize_a_package.3F

Reply:
That wiki page is not really useful for our purposes. Maybe we could use the [mailing list announcement](https://archives.gentoo.org/gentoo-dev-announce/message/3b1610ae48204aed5552fca4b4c6c135)?

(I have nothing against linking to github, but the fragment identifier there is only working with javascript. Also I had to fix too many broken external links during the last months. :smile:)

Reply:
Oh, Github is one of few sites where I have javascript enabled.

Sure, changed the link, and rephrased this - from reading that link one can assume that they need to run some command for it to work.

PTAL

Reply:
Oh, that thing. The docs are also available at `https://dev.gentoo.org/~mgorny/doc/nattka/`

Reply:
> Oh, that thing. The docs are also available at `https://dev.gentoo.org/~mgorny/doc/nattka/`

Yes, let's use that. And no fragment identifiers please, because they are prone to future breakage.

Reply:
Done